wireless_modules/middle_module/MQTT_Client_class.py

```diff
@@ -0,0 +1,102 @@
+# Author: Kunj (most is also borrowed from mqtt_test written by Blake)
+# Last modified: 3/08/2020
+# Note: Can remove all the Print statements once all testing is successful
+
+# Use micro pip to install packages
+import utime
+import upip
+upip.install("micropython-umqtt.simple2")
+try:
+    from umqtt.simple import MQTTClient
+except FileNotFoundError:
+    print("Error in importing library")
+
+
+class Client:
+    def __init__(self, client_id, mqtt_broker):
+        """
```

Comment:
Great documentation! :+1:

Reply:
I really like this documentation style. Is this some sort of docstring convention?

Reply:
Yeah pretty sure it is a convention for docstring